liquibase-extension-testing/src/main/java/liquibase/extension/testing/testsystem/TestSystemFactory.java

```diff
@@ -67,8 +67,9 @@ public TestSystem getTestSystem(TestSystem.Definition definition) {
 
     private List<String> getConfiguredTestSystems() {
         final String testSystems = Scope.getCurrentScope().getSingleton(LiquibaseConfiguration.class).getCurrentConfiguredValue(ConfigurationValueConverter.STRING, null, "liquibase.sdk.testSystem.test").getValue();
+        final String skippedTestSystems = Scope.getCurrentScope().getSingleton(LiquibaseConfiguration.class).getCurrentConfiguredValue(ConfigurationValueConverter.STRING, null, "liquibase.sdk.testSystem.skip").getValue();
 
-        return CollectionUtil.createIfNull(StringUtil.splitAndTrim(testSystems, ","));
+        return TestSystem.determineEnabledTestSystems(testSystems, skippedTestSystems, getTestSystemNames());
```

Comment:
The fact that we're now having to call a static method on TestSystem from the factory is maybe showing that it's time to move the "enabled test systems" logic from static TestSystem logic to TestSystemFactory? 

Also, maybe better to have the method name `getEnabledTestSystems()` ?

Reply:
I'm not going to do the refactor you're describing, despite it being time for it, since I don't have time to do that now.

I renamed the method.